### DISCUSSION TITLE
May you help to add a property hide/show option in setting? Thanks!

### DISCUSSION THREAD

**Discussion by nanjingman** [tu2-atmanand/cards-view-plugin-beta#16] [Q&A]
May you help to add a property hide/show option in setting?  I want to hide the properties in the cardsview. Thanks!

![image](https://github.com/user-attachments/assets/0db8ce8a-7aa5-4387-83f4-152c43acfa2b)

**Comment by tu2-atmanand** [comment]
Hii @nanjingman !

Glad you like the feature I have implemented in Notes Explorere. Unfortunately, I dont have direct access to modify the Cards View plugin. Still I have been in a process of contineous contribution process with the Cards view. I have submitted my work to the author of Cards view plugin, but again, its all upto the author to accept my changes. In Open-source arena it takes a long time to collaborate and integrate new changes. So, I cant forsee when this feature will be integrated in the Cards view plugin. But, you are free to use this plugin by installing through the process I have mentioned in the Readme, its safe to install and you can enjoy the new features.

Always, interested in hearing new suggestions from my user. If you like me to add any new feature in the Notes explorer plugin, feel free to create a new issue and explain your feature/suggestion, and ill try my best to implement it.

BTW, I have almost finished with the biggest optimization I was working on Notes Explorer, Ill be posting it in this topic : https://forum.obsidian.md/t/notes-explorer-fork-of-cards-view-plugin/93135?u=tu2_atmanand
